src/webrtc/call.js

```diff
@@ -753,14 +767,26 @@ MatrixCall.prototype._gotLocalOffer = function(description) {
             },
             lifetime: MatrixCall.CALL_TIMEOUT_MS,
         };
-        sendEvent(self, 'm.call.invite', content);
+        sendEvent(self, 'm.call.invite', content).then(() => {
+            setState(self, 'invite_sent');
+            setTimeout(function() {
+                if (self.state == 'invite_sent') {
+                    self.hangup('invite_timeout');
+                }
+            }, MatrixCall.CALL_TIMEOUT_MS);
+        }).catch((error) => {
+            self.client.cancelPendingEvent(error.event);
+            terminate(self, "local", "unknown_devices", false);
+            self.emit(
```

Comment:
again, surely we should check the type of error before assuming it's an Unknown Devices?

Reply:
fixed here too